src/main/scala/dotty/dokka/tasty/BasicSupport.scala

```diff
@@ -64,7 +64,9 @@ class SymOps[R <: Reflection](val r: R) {
         Option.when(sym.flags.is(Flags.Case))(ScalaOnlyModifiers.Case)
       ).flatten
 
-    def shouldDocumentClasslike: Boolean = !sym.flags.is(Flags.Private) && !sym.flags.is(Flags.Synthetic)
+    def shouldDocumentClasslike: Boolean = !sym.flags.is(Flags.Private) && !sym.flags.is(Flags.Synthetic) && (!sym.flags.is(Flags.Case) || !sym.flags.is(Flags.Enum))
```

Comment:
This looks too restrictive based on a quick look into code. We can have the following:

```scala
class Clazz {
  sealed trait Foo
  case class Bar() extends Foo
}
```

```scala
object Obj {
  sealed trait Foo; case class Bar extends Foo
}
```

And we should list `Foo`, `Bar` among the members of `Clazz` and `Obj`. Would this check prevent that? We also can have `enum`-s nested inside classes and objects.

Reply:
Hmm but the condition is that it can't be Case and Enum **together**, so in both cases Foo and Bar will be documented, I think.